src/form/state/FormFieldInputState.test.ts

```diff
@@ -0,0 +1,94 @@
+import { createDefaultState } from './FormFieldDefaultState';
+import { createInputState } from './FormFieldInputState';
+import { FormFieldInputState } from './FormFieldInputState.model';
+
+describe('Form field input state', () => {
+  describe('value', () => {
+    it('is initially empty', () => {
```

Comment:
Can we make this `is empty by default` instead as you can actually predefine a value while initiating?